app/config/AppConfig.scala

```diff
@@ -81,4 +81,12 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
     agentClientLookupHost +
       servicesConfig.getConfString(vatAgentClientLookupFrontendStartUrl, "") +
       s"?redirectUrl=${agentClientLookupRedirectUrl(uri)}"
+
+  lazy val pegaBearerToken: String = "bearerToken" //TODO: bearerToken to be configured for PEGA
```

Comment:
Similar to the layout above, we tend to add the key to application.conf and assign a value to it. So in application.conf have a new key-value:
```
pega {
    bearerToken = "placeholder"
}
```

Then you can replace this with `config.get[String]("pega.bearerToken")`

Reply:
Ok, then it seems the bearer token is not getting it from generated from any Authentication Server.

Reply:
The bearer token obviously won't change anyway. We will be provided with the bearer token later so we can pass it with each request. By having it in application.conf makes it easier to change the bearer token without having to release a new version of the service.